mmaction/models/heads/tsm_head.py

```diff
@@ -34,8 +34,9 @@ def __init__(self,
                  dropout_ratio=0.8,
                  init_std=0.001,
                  is_shift=True,
+                 multi_class=False,
```

Comment:
Please add description for this args in the docstring like [this](https://github.com/open-mmlab/mmaction2/blob/master/mmaction/models/heads/base.py#L40).